ceph.spec.in

```diff
@@ -159,7 +164,17 @@ BuildRequires:	gperftools-devel >= 2.6.1
 BuildRequires:	gperftools-devel >= 2.4
 %endif
 %endif
+%if 0%{with make_check}
 BuildRequires:  jq
+%endif
```

Comment:
Please merge this make_check conditional block into the big make_check conditional block, below.

Reply:
done.